proto/cmp/services/book/v1alpha/mint.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package cmp.services.book.v1alpha;
+
+import "cmp/types/v1alpha/common.proto";
+import "cmp/types/v1alpha/currency.proto";
+import "cmp/types/v1alpha/language.proto";
+import "cmp/types/v1alpha/payment.proto";
+import "cmp/types/v1alpha/pubkey.proto";
+import "cmp/types/v1alpha/traveller.proto";
+import "google/protobuf/timestamp.proto";
+
+message MintRequest {
+  // Message header
+  cmp.types.v1alpha.Header header = 1;
+
+  string validation_id = 2;
+
+  string external_session_id = 3;
+
+  cmp.types.v1alpha.Currency currency = 4;
+
+  cmp.types.v1alpha.Language language = 5;
+
+  string market = 6;
+
+  string booking_reference = 7;
+
+  repeated cmp.types.v1alpha.ExtensiveTraveller travellers = 8;
+
+  string comment = 9;
```

Comment:
What is the purpose of a comment in an automated request which is most likely not being reviewed by a human?

Reply:
Decided to keep it after discussions. Comment is for information that is not obligatory, only a "nice to have" features that the buyer requests from the service provider, ex: hotel.